<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BannedIPDialog</class>
 <widget class="QDialog" name="BannedIPDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ban Manager</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>181</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancelbtn">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="lb_bannedIPs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_IP">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddIPbtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add IP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
